galsim/zernike.py

```diff
@@ -18,8 +18,9 @@
 
 import numpy as np
 
-from .utilities import LRU_Cache, binomial, horner2d, nCr, lazy_property
-from .errors import GalSimValueError, GalSimRangeError
+from .utilities import LRU_Cache, binomial, horner2d, horner4d, nCr, lazy_property
+from .integ import _gq_annulus_points
```

Comment:
Why the private function rather than the `gg_annulus` function you made?  If you feel the need to use this undocumented function instead, that strikes me as a sign that the front end function with nice documentation wasn't the right API.  Probably worth rethinking this a bit so that this file can use the documented function.  Or possibly just promote this function to a non-leading_underscore function with better docs.

Reply:
Decided to make this one public (and add docs).